locallib.php

```diff
@@ -3660,5 +3680,41 @@ function bigbluebuttonbn_create_meeting_metadata(&$bbbsession) {
     if ((boolean) \mod_bigbluebuttonbn\locallib\config::get('meetingevents_enabled')) {
         $metadata['analytics-callback-url'] = $bbbsession['meetingEventsURL'];
     }
+    // Special metadata for Opencast recordings (passing opencast seriesid of the course as opencast-dc-isPartOf as metadata).
```

Comment:
I would propose to add the constraint that the OC plugin has to be installed and the option has to be enabled here once more:
```suggestion
    // If block_opencast is installed and the option to send the Opencast series ID to BBB is enabled,
    // pass the Opencast series ID of the course as opencast-dc-isPartOf within the BBB metadata.
```